docs/intro/overview.txt

```diff
@@ -40,7 +40,7 @@ database-schema problems. Here's a quick example:
         pub_date = models.DateField()
         headline = models.CharField(max_length=200)
         content = models.TextField()
-        reporter = models.ForeignKey(Reporter)
+        reporter = models.ForeignKey(Reporter, null=True, on_delete=models.SET_NULL)
```

Comment:
Should we add `blank=True` wherever we have `null=True`? Longer lines should be reformatted:
```python
reporter = models.ForeignKey(
    Reporter,
    blank=True,
    null=True,
    on_delete=models.SET_NULL,
)
```